renovte.json

```diff
@@ -0,0 +1,40 @@
+{
+  "pinVersions": false,
+  "semanticCommits": true,
+  "semanticPrefix": "build",
+  "separateMajorMinor": false,
+  "prHourlyLimit": 2,
+  "labels": ["comp: build", "dependencies"],
+  "timezone": "America/Tijuana",
+  "lockFileMaintenance": {
+    "enabled": true
+  },
+  "schedule": ["after 10pm every weekday", "before 4am every weekday", "every weekend"],
+  "baseBranches": ["main"],
+  "ignoreDeps": ["@types/node", "@types/selenium-webdriver", "selenium-webdriver"],
```

Comment:
Is there context on these ignore packages?

Reply:
-`selenium-webdriver` version 4 beta which is current marked as latest.
- `@types/node` seems to be pinned to the min supported version fo the Angular framework and hence the update needs to happen manually.

Reply:
👍 thanks. good to capture this here.